### ISSUE TITLE
Controller - Move DocumentController in EditorController

### ISSUE THREAD

**Issue by adrian-moisa** [visual-space/visual-editor#222]
The article subscribes to the initialization process and receives the default synchronized article. Now, it wants to update the VE (Visual Editor) controller. To perform the update, it needs the documentController. The documentController is created during the initialization of the VE widget. The VE widget is created when the article page triggers the build process, including the articleOk state store emit (the second emitted state). The build process, along with the visible VE, occurs after the article subscription initially emits the default values in sync. In other words, we are trying to update the documentController that the VE internally assumes to be available, even though we haven't created the VE widget yet, which creates the document controller.

In short, we want to update a document using a document controller before the document controller is created.

**Comment by adrian-moisa** [comment]
I believe this was fixed in #228
